pkgs/development/ocaml-modules/uuidm/default.nix

```
@@ -1,5 +1,8 @@
 { lib, stdenv, fetchurl, ocaml, findlib, ocamlbuild, topkg, cmdliner }:
 
+lib.throwIfNot (lib.versionAtLeast ocaml.version "4.08")
+  "uuidm is not available for OCaml ${ocaml.version}"
+
```

not related to this change but still valid.
```suggestion
lib.throwIf (lib.versionOlder ocaml.version "4.08")
  "uuidm is not available for OCaml ${ocaml.version}"
```
Avoid double negation ?